schulcloud/LoginViewController.swift

```diff
@@ -25,6 +25,10 @@ class LoginViewController: UIViewController {
         usernameTextArea.delegate = self
         passwordTextArea.delegate = self
         usernameTextArea.text = defaults.string(forKey: usernameKey)
+        
+        DispatchQueue.global(qos: .default).sync {
+            recreatePersistentContainer()
+        }
```

Comment:
Fair enough, I will change this in the coming days. Additionally, I'll remove the second instantiation of the login view controller